namecmds.go

```diff
@@ -56,6 +56,17 @@ func NewNameShowCmd(name string, options *NameShowOptions) *NameShowCmd {
 	}
 }
 
+// NameListCmd defines the name_list JSON-RPC command.
+type NameListCmd struct {
+	Name string `json:"name,omitempty"`
```

Comment:
Hmm, elsewhere in this file we don't use the `json` annotations for command definitions.  Is there a precedent for doing this in upstream btcd?